src/PhpSpec/Formatter/Presenter/StringPresenter.php

```diff
@@ -359,4 +366,75 @@ private function presentCallable($value)
 
         return sprintf('[%s()]', $value);
     }
+
+    /**
+     * @param UnexpectedCallException $exception
+     *
+     * @return string
+     */
+    private function presentCallArgumentsDifference(UnexpectedCallException $exception)
+    {
+        $text = '';
+
+        $actualArguments = $exception->getArguments();
+        $methodProphecies = $exception->getObjectProphecy()->getMethodProphecies($exception->getMethodName());
+        $presentedMethodProphecy = $this->findMethodProphecyOfFirstNotExpectedCall($methodProphecies, $exception);
+        $expectedTokens = $presentedMethodProphecy->getArgumentsWildcard()->getTokens();
+
+        if (count($expectedTokens) !== count($actualArguments)) {
+
+            //parameters count mismatch will be shown in exception message, no need in diff
```

Comment:
I'd rather not have inline comments, can you try to make it explicit from the code? Extracting the conditional to a method might help in this instance.